onnxruntime/core/providers/cpu/cpu_provider_factory.cc

```diff
@@ -15,7 +15,7 @@ struct CpuProviderFactory {
   CpuProviderFactory();
 };
 
-ONNXStatus* ONNXRUNTIME_API_STATUSCALL CreateCpu(void* this_, ONNXRuntimeProvider** out) {
```

Comment:
this is why it was called ...STATUSCALL... to differentiate from the regular ONNXRUNTIME_API_CALL invocations.

Reply:
had offline conversation with Ryan. this is ok.